OpenSourcePolitics/decidim-cese
# Droits admins petition

#### :tophat: Description
When publishing a petition, this pr adds the petition's blog admin rights to the petition author.
It also removes these when the petition is unpublished.

It also fixes the 2 following error : 
- When you are a scoped admin there was an error when publishing a post
- when you create a new scoped admin constraint on an initiative, there were no components you could select

#### :pushpin: Related Issues
- [Notion card](https://www.notion.so/opensourcepolitics/P-titions-CESE-Droits-admin-sur-les-actualit-s-d-une-p-tition-au-mandataire-ea788c49ad6b489da8adcd0cc116927c?d=716b9e907c904c0cbf5d749bc027aca7)

#### Testing
* Log in as user
* Create a petition and send it to technical validation
* Log out and log in as admin
* Publish the petition
* Go to the scope admin section in awesome to see the rights have been added
* Log out and log in as user
* See you have admin rights on the petition blog
* Log out and log in as admin
* Unpublish the petition
* Go to the scope admin section in awesome to see the rights have been removed
* Log out and log in as user
* See you no longer have admin rights on the petition blog

For the first error

* Log in as scoped admin and publish the post

For the second error
* Log in as admin
* Go to scoped admin tab in awesome
* Add a new contraint and dee you can select specific component in the last select

#### Tasks
